se/se_epub_lint.py

```diff
@@ -1134,9 +1135,28 @@ def lint(self, skip_lint_ignore: bool) -> list:
 				xml_dom = self.get_dom(filename)
 
 				# / selects the root element, so we have to test against the name instead of doing /search-key-map
-				if xml_dom.xpath("/search-key-map") and filename.name !="glossary-search-key-map.xml":
+				if xml_dom.xpath("/search-key-map") and filename.name != "glossary-search-key-map.xml":
 					messages.append(LintMessage("f-013", "Glossary search key map must be named [path]glossary-search-key-map.xml[/].", se.MESSAGE_TYPE_ERROR, filename))
 
+				# Make sure that everything in glossaries are in the rest of the text
+				if filename.name == "glossary-search-key-map.xml":
+					# Map the glossary to tuples of the values and whether they’re used (initially false)
+					glossary_usage = list(map(lambda node: (node.get_attr("value"), False), xml_dom.xpath(".//*[@value]")))
+					# Walk the tree and mark if any of the glossary entries are used
+					for text_root, text_directories, text_filenames in os.walk(os.path.join(self.path, "src", "epub", "text")):
```

Comment:
Can you use Pathlib to form paths instead of os.path.join? E.g. `self.path / "src/epub/text"`

Also, we should do a recursive walk over the entire epub content path to match xhtml files, in case in the future there are some files that are not in `./text/`. See elsewhere in lint for the `self.content_path.glob("**/*.xhtml")` pattern